PyU4V/replication.py

```diff
@@ -1062,3 +1066,267 @@ def delete_storage_group_srdf(
+        run get_rdf_remote_port_details and create the initial connection
+        group using only the first source port you can extract details for one
+        or more remote ports for the desired target array and feed into this
+        function. Additional Ports can be added with modify_rdf_group function.
+
+        :param local_director_Port_list: list of local directors and ports for
+                                        group e.g [RF-1E:1, RF-2E:1] -- list
+        :param remote_array_id: 12 digit serial number of remote array  -- str
+        :param label: Label for group up to 10 characters -- str
+        :param local_rdfg_number: rdfg for the local array-- int
+        :param remote_rdfg_number rdfg for the remote array -- int
+        :param remote_director_port_list: list of remote directors and ports to
+                                          group e.g [RF-1E:1, RF-2E:1] -- list
+        :param array_id: array_id: 12 digit serial number of Source (R1) array,
+                        if no array is specified the array in config file or
+                        api connection will be default -- str
+        """
+        if not array_id:
+            array_id = self.array_id
+        local_ports = []
+        remote_ports = []
+        for port in local_director_port_list:
+            dir_id = port.split(':')[0]
+            port_no = port.split(':')[1]
+            local_ports.append({'symmetrixID': array_id,
+                                'directorId': dir_id,
+                                'portNumber': port_no})
+        for port in remote_director_port_list:
+            dir_id = port.split(':')[0]
+            port_no = port.split(':')[1]
+            remote_ports.append({'symmetrixID': remote_array_id,
+                                 'directorId': dir_id, 'portNumber': port_no})
+
+        payload = {
+            'local_ports': local_ports,
+            'remote_rdfg_number': remote_rdfg_number,
+            'remote_ports': remote_ports,
+            'label': label,
+            'local_rdfg_number': local_rdfg_number}
+
+        self.create_resource(category=REPLICATION,
+                             resource_level=SYMMETRIX,
+                             resource_level_id=array_id,
+                             resource_type=RDF_GROUP, payload=payload)
+
+    def modify_rdf_group(self, action, srdf_group_number, array_id=None,
+                         port_list=None, label=None, dev_list=None,
+                         target_rdf_group=None, consistency_exempt=None):
+        """Function to Modify Ports, devices or change label of RDF group.
+
+        Function can be used to Add ports, move volumes between rdf groups,
+        remove ports or rename RDF group.
+
+        :param action: add_ports, remove_ports, move
+        :param srdf_group_number: srdf group number
+                                  for action on local array: int
```

Comment:
not sure if a second line is needed here, and -- should be used instead of : when defining input type